packages/sdk/client-services/src/packlets/tests/client-services.test.ts

```diff
@@ -213,7 +213,7 @@ describe('Client services', () => {
 
     const space1 = await client1.createSpace();
     log('createSpace', { key: space1.key });
-    const observable1 = space1.createInvitation({ type: Invitation.Type.INTERACTIVE_TESTING });
+    const observable1 = space1.createInvitation({ authMethod: Invitation.AuthMethod.NONE });
```

Comment:
per previous PR, authenticationMethod (just consistency with other params?)
Esp auth since "authent" and "author" are often confused

Reply:
On second though, since this is everywhere just consider shortening authenticationCode to authCode. just make symmetrical